### PR TITLE
Add [machine-learning] to [machine-learning]

### LINKED ISSUE #15: Docs: Create machine learning resources folder
I would love to contribute by creating a machine learning resources folder. 

Kindly assign this to me

### PR DESCRIPTION
**Pull request by motuncoded**
This folder contains machine learning resources 
closes #15

### CODE REVIEW

File: MachineLearning/MachineLearning.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+![Machine Learning](./assets/machinelearning.jpg)
+
+## 📑 Table of Contents
+1. [Beginner Level](#-beginner-level)
+2. [Intermediate Level](#-intermediate-level)
+3. [Advanced Level](#-advanced-level)
+4. [Popular Tools & frameworks](#-popular-tools-and-frameworks)
+5. [Additional Resources](#-additional-resources)
+6. [Contribute](#-contribute)
```
Comment (Charity10) [inline MachineLearning/MachineLearning.md:13]: Should be 'contributing' or you change the heading below to 'contribute'

File: MachineLearning/MachineLearning.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### Advanced Resources:
+1. **[Deep Learning Specialization (Coursera)](https://www.coursera.org/specializations/deep-learning)** - A complete course on deep learning from Andrew Ng.
+2. **[Advanced Machine Learning on Kaggle](https://www.kaggle.com/learn/advanced-machine-learning)** - Learn how to work with time series, natural language, and deep learning.
+3. **[A/B Testing in Machine Learning](https://medium.com/swlh/a-b-testing-in-machine-learning-107d9b3a0059)** - Techniques for evaluating model performance in production.
+4. **[MLFlow for Model Management](https://mlflow.org/)** - A tool for managing machine learning models, experiments, and deployments.
+
+---
+
+## ⚒️ Popular Tools & Frameworks
```
Comment (Charity10) [inline MachineLearning/MachineLearning.md:63]: If you remove the 'and' in the table of content and the link is still not working, try replacing the emoji used here

**Review by Charity10 — CHANGES_REQUESTED** [review]
Lovely work done so far 👏
for some strange reason the images aren't showing but just make the changes requested so I can merge and see if they will show up after being merged.

**Comment by motuncoded** [comment]
Thanks for the changes, would be glad to correct them as soon as possible.

**Pull request merged**